doc/release/upcoming_changes/12284.new_feature.rst

```diff
@@ -0,0 +1,6 @@
+
+Add our own ``*.pxd`` cython import file
+--------------------------------------------
+Added a ``numpy/__init__.pxd`` file. It will be used for `cpython import numpy`
```

Comment:
```suggestion
Added a ``numpy/__init__.pxd`` file. It will be used for `cimport numpy as npc`
```
is what you mean I think? And I am not sure we still need the search path link (at least not as a link).

Reply:
fixed, thanks